src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -595,7 +597,16 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
         cryptoInputValidation={cryptoInputValidation}
         fiatAmountAvailable={fiatAmountAvailable.toString()}
         fiatInputValidation={fiatInputValidation}
-        marketData={assetMarketData}
+        marketData={
+          assetMarketData ?? {
+            price: '0',
+            marketCap: '0',
+            volume: '0',
+            changePercent24Hr: 0,
+            supply: '0',
+            maxSupply: '0',
+          }
```

Comment:
We probably want to have an empty market data constant, as this pattern appears often in this PR.